axi_tdd: Add false paths to tdd sync input

Yamakaja opened the pull request:
This allows the external synchronization input to be driven from
asynchronous sources like a 1 PPS signal or just signals from different
clock domains in general.

Yamakaja commented:
Generally i do agree that moving synchronization logic to util_tdd_sync would make sense, but that introduces a different issue:

If you try to use the TDD engine in an environment with variable clocks (Like the AD9361 sample clock), you will run into the situation where the pulse generator in the util_tdd_sync would have to be retuned based on the clock rate. The current solution to that issue is to run the pulse gen at a constant clock rate and then introduce a CDC in the axi_tdd.

Any suggestions as to how that could be addressed?

ronagyl commented:
I see,  in this case it would be nice to have the CDC optional in the axi_tdd,  with a synthesis parameter, on by default, and ttcl file for the xdc.  

For example in case of  MxFE the util_tdd_sync  runs at the same clock as the axi_tdd.  In such case I would disable the CDC,  just for the sake of a clean clock interaction report.

Yamakaja commented:
Alright, i see how that would be useful - i've implemented something similar for the data offloads external sync signal ;)

Yamakaja commented:
@ronagyl I've implemented your suggestion and rebased onto current master. This PR now also contains a commit to disable the synchronization stage for the ad9081 project.
